release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3631,7 +3631,7 @@ $ oc adm release info 4.14.34 --pullspecs
 [id="ocp-4-14-34-known-issue_{context}"]
 ==== Known issues
 
-* On clusters with the SR-IOV Network Operator installed and configured, pods with a secondary interface of SRI-OV VF fail to create a pod sandbox and will not function. (link:https://issues.redhat.com/browse/OCPBUGS-38091[*OCPBUGS-38091*])
+* On clusters with the SR-IOV Network Operator installed and configured, pods with a secondary interface of SR-IOV VF fail with the error message: `SRIOV-CNI failed to configure VF "failed t │ o set vf 0 vlan configuration - id 0, qos 0 and proto 802.1q: invalid argument"`. To resolve this issue, upgrade the SR-IOV Network Operator before upgrading {product-title}. This ensures that the fix to this issue is included in the SR-IOV Network Operator. (link:https://issues.redhat.com/browse/OCPBUGS-38091[*OCPBUGS-38091*])
```

Comment:
`failed t │ o set vf` -> `failed to set vf`

Reply:
Fixed, thanks!